src/category_theory/adjunction/limits.lean

```diff
@@ -32,15 +32,17 @@ open category_theory
 open category_theory.functor
 open category_theory.limits
 
-universes u₁ u₂ v
+universes u₁ u₂ v₁ v₂ v v₀
 
-variables {C : Type u₁} [category.{v} C] {D : Type u₂} [category.{v} D]
+section arbitrary_universe
+
+variables {C : Type u₁} [category.{v₁} C] {D : Type u₂} [category.{v₂} D]
 
 variables {F : C ⥤ D} {G : D ⥤ C} (adj : F ⊣ G)
 include adj
 
 section preservation_colimits
-variables {J : Type v} [small_category J] (K : J ⥤ C)
+variables {J : Type v} [category J] (K : J ⥤ C)
```

Comment:
Could you please explicitly name the universe variable. The benefit is small, but nice to have: declarations will have a nice universe name instead of `u_1` that is "competing" with `u₁`.
```suggestion
variables {J : Type v} [category.{put something here} J] (K : J ⥤ C)
```

Reply:
Thanks for the comment; will do. The PR may not be ready for a while as at least two other files are broken ...

Reply:
There are still some universe variables that are not explicitly named. I think its okay to not do so in those places where the universe can be inferred from the arguments, but I'm just pointing it out in case Johan thinks otherwise.